src/DatabaseGpt.SqlServer/SqlServerDatabaseGptProvider.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Data;
+using System.Text;
+using Dapper;
+using DatabaseGpt.Abstractions;
+using DatabaseGpt.Abstractions.Exceptions;
+using Microsoft.Data.SqlClient;
+
+namespace DatabaseGpt.SqlServer;
+
+public class SqlServerDatabaseGptProvider : IDatabaseGptProvider, IDisposable
+{
+    private readonly SqlServerDatabaseGptProviderConfiguration settings;
+    private bool disposedValue;
+
+    public SqlServerDatabaseGptProvider(SqlServerDatabaseGptProviderConfiguration settings)
+    {
+        this.settings = settings;
+        this.connection = new SqlConnection(settings.ConnectionString);
+        connection.Open();
+    }
+
+    private readonly SqlConnection connection;
+
+    public string Name => "SQL Server";
+    public string Language => "T-SQL";
+
+    public async Task<IEnumerable<string>> GetTablesAsync(IEnumerable<string> excludedTables)
+    {
+        var tables = await connection.QueryAsync<string>("SELECT TABLE_SCHEMA + '.' + TABLE_NAME AS Tables FROM INFORMATION_SCHEMA.TABLES;");
+        return tables.Where(t => !excludedTables.Contains(t));
```

Comment:
Add a blank line before `return` statement. Add the support for included tables, as in https://github.com/marcominerva/DatabaseGPT/blob/master/src/DatabaseGpt/DatabaseGptClient.cs#L117-L131.